config/environments/test.rb

```diff
@@ -25,19 +27,19 @@
 
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
+  config.action_mailer.perform_caching = false
 
   # Tell Action Mailer not to deliver emails to the real world.
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
-  # Randomize the order test cases are executed.
-  config.active_support.test_order = :random
```

Comment:
To clarify what we just talked about, this setting is being removed here because this is the new default behavior. There will be no change in behavior by removing this line.